doc/Type/Metamodel/Stashing.pod6

```diff
@@ -0,0 +1,61 @@
+=begin pod :kind<Type> :subkind<role> :category<metamodel>
+
+=TITLE role Metamodel::Stashing
+
+=SUBTITLE Metarole for type stashes
+
+    role Metamodel::Stashing { }
+
+Types may have a stash associated with them. For types where this is
```

Comment:
Can you please clarify the concept of stash? For instance, it's a "place" to store something temporarily.